apps/s_server.c

```diff
@@ -2606,6 +2606,15 @@ static void close_accept_socket(void)
     }
 }
 
+static int is_retryable(SSL *con, int i)
+{
+    int err = SSL_get_error(con, i);
+
+    /* If it's not a fatal error, it must be retryable */
+    return (err != SSL_ERROR_SSL) && (err != SSL_ERROR_SYSCALL)
```

Comment:
Oy, how `util/openssl-format-source` is gonna change things when / if we run it again...

Not complaining about the result here, but let's be aware how we change the coding style be sheer habits.